pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -126,6 +127,20 @@ type QueueSortPlugin interface {
 	Less(*PodInfo, *PodInfo) bool
 }
 
+// FailedPredicateMap declares a map[string][]algorithm.PredicateFailureReason type.
+type FailedPredicateMap map[string][]predicates.PredicateFailureReason
```

Comment:
Let's wait for #78477 to merge. I believe we should use [`FailedFilterMap`](https://github.com/kubernetes/kubernetes/pull/78477/files#diff-2360941f343e5173db49c10fe081c008R211) here instead of `FailedPredicateMap` as the type of the argument that we pass to `PostFilter` plugins.
We should write a function that converts `FailedPredicateMap` returned by `findNodesThatFit` to a `FailedFilterMap`.

Reply:
OK, I'll keep an eye on #78477